src/chunktransformer.ts

```diff
@@ -14,11 +14,46 @@
  * limitations under the License.
  */
 import {Transform, TransformOptions} from 'stream';
-
 import {Mutation} from './mutation';
 
+export type Next = () => void;
+
+export type ValueOrBuffer = Value|Buffer;
+export type Value = string|number|boolean|Uint8Array;
+
+export interface Chunk {
+  rowContents: Value;
+  commitRow: boolean;
+  resetRow: boolean;
+  rowKey?: string|Uint8Array;
+  familyName?: {value: string};
+  qualifier?: Qualifier|{value: Value};
+  timestampMicros?: number|Long;
+  labels?: string[];
+  value?: ValueOrBuffer;
```

Comment:
I think this will either be a string or a buffer

Reply:
Yeah, `valueOrBuffer` smelled funny the entire time. I should probably listen to that feeling more. Sourced it from `Mutation.convertFromBytes()` logic. Fixed 👍